src/dependabot/update_metadata.ts

```diff
@@ -110,6 +116,35 @@ export async function parse (commitMessage: string, body: string, branchName: st
   return Promise.resolve([])
 }
 
+/**
+ * Parses the human-readable metadata links from a commit message.
+ * <p>
+ *    See {@code Dependabot::PullRequestCreator::MessageBuilder#metadata_links} in the Ruby codebase for more details
+ *    on the current format.
+ * </p>
+ * <p>
+ *     <strong>Note:</strong> This data is only available if more than one dependency is updated in a single PR.
+ * </>
```

Comment:
I love that there's a detailed comment, but why the html tags? Is there something about typescript tag formats that I'm unaware of?

Reply:
I am not a TypeScript developer, so I used JavaDoc (HTML-based) instead of JSDoc.

https://github.com/dependabot/fetch-metadata/pull/632/commits/a6302c97954432d25357877878d466d6abc8ad7b should fix this.